infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -50,18 +66,15 @@
      * actual threshold would abort due to NEW_SEQUENCER cause.
      */
     @Getter
-    private long maxConflictNewSequencer = Address.NOT_FOUND;
-
-    @VisibleForTesting
-    public SequencerServerCache(long cacheSize, CacheWriter<ConflictTxStream, Long> writer) {
-        this.conflictCache = Caffeine.newBuilder()
-                .maximumSize(cacheSize)
-                .writer(writer)
-                //Performing periodic maintenance using current thread (synchronously)
-                .executor(Runnable::run)
-                .recordStats()
-                .build();
-    }
+    private long maxConflictNewSequencer;
+
+    /* It is used to calculate the size of ServerCache.
+     * Each entry relates two pointers used by HashMap, one pointer in PriorityQueue.
+     */
+    static final int Entry_Overhead = 24;
```

Comment:
please rename according to code convention
also if it possible make it private

Reply:
Done.